text/java/0012-quarkus.md

```diff
@@ -0,0 +1,63 @@
+# Quarkus Buildpack
+
+## Summary
+Provide a Quarkus Buildpack which will be part of the order groups for `paketo-buildpacks/java` and `paketo-buildpacks/java-native-image`.
+The primary purpose of this buildpack is to set environment variables for the `paketo-buildpacks/maven` and `paketo-buildpacks/native-image` buildpacks. Building a Quarkus application is presently possible but requires setting multiple environment variables to properly configure the buildpacks. This proposed buildpack will enable users to build Quarkus applications without additional configuration.
+
+Proof of concept could be found at: [quarkus-bp][b]
+
+[b]: https://github.com/matejvasek/quarkus-bp
+
+## Motivation
+While building Quarkus application is already possible it's unnecessarily complicated.
+The developer have to set 2 to 4 environment variables which is not obvious.
+It should be as easy as just running: 
+```sh
+pack build my-img -B quay.io/mvasek/builder:base --env BP_NATIVE_IMAGE=1
+```
+instead of:
+```sh
+pack build my-img -B quay.io/mvasek/builder:base \
+  --env BP_NATIVE_IMAGE=1 \
+  --env "BP_MAVEN_BUILD_ARGUMENTS=package -DskipTests=true -Dmaven.javadoc.skip=true -Dquarkus.package.type=native-sources" \
+  --env "BP_MAVEN_BUILT_ARTIFACT=target/native-sources/*" \
+  --env "BP_NATIVE_IMAGE_BUILD_ARGUMENTS_FILE=native-image.args" \
+  --env "BP_NATIVE_IMAGE_BUILT_ARTIFACT=*-runner.jar"
+
+```
+
+## Detailed Explanation
+The Quarkus Buildpack will be moved into the paketo-buildpacks Github organization and added to the Java sub-team. Maintenance for this buildpack will be provided by the Java subteam.
+
+The following changes will be made:
+* [Quarkus Buildpack][b] moved to Paketo Buildpacks org
+* The Buildpack will be published to gcr.io/paketo-buildpacks/quarkus
+* The Buildpack will be added to the `paketo-buildpacks/java` and `paketo-buildpacks/java-native-image` buildpacks groups just before the `paketo-buildpacks/maven` buildpack.
+
```

Comment:
I would like to see what the proposed order group is for `paketo-buildpacks/java` (the others should be basically the same).